pkg/controller/replication/replication_controller.go

```diff
@@ -660,13 +663,13 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 			return aggregate
 		}
 	} else {
-		podList, err := rm.podStore.Pods(rc.Namespace).List(labels.Set(rc.Spec.Selector).AsSelector())
+		pods, err := rm.podStore.Pods(rc.Namespace).List(labels.Set(rc.Spec.Selector).AsSelector())
```

Comment:
It is - but it seems significantly worse with GC enabled (I think it's related to other loops through those pods).